src/test/test-cgroup.c

```diff
@@ -75,19 +76,19 @@ TEST(cg_create) {
 
         assert_se(cg_pid_get_path(SYSTEMD_CGROUP_CONTROLLER, getpid_cached(), &path) == 0);
         assert_se(streq(path, test_b));
-        free(path);
+        path = mfree(path);
```

Comment:
Hm, this is not bad, but if you change this, then please change the declaration of `path` as `_cleanup_free_ char *path = NULL;`. This is much safer.